toolchain/ltrace.py

```diff
@@ -58,7 +58,7 @@ def setUp(self):
         elif dist_name in ("rhel", "fedora", "redhat"):
             packages.extend(['elfutils-devel', 'elfutils-libelf-devel', 'git',
                              'elfutils-libelf', 'elfutils-libs',
-                             'libtool-ltdl'])
+                             'libtool', 'libtool-ltdl'])
```

Comment:
@preetithakur will this work for all the version of RHEL? or this is for latest distro version of RHEL?
if the requirement of this package is from latest distro version of RHEL and not required for older versions then please add the logic for it, so that it will pick for only those distro versions where it is needed. and doesn't breaks for older versions.